packages/react-conformance-griffel/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-conformance-griffel",
-  "version": "9.0.0-beta.0",
+  "version": "9.0.0-rc.0",
```

Comment:
Can we wait on bumping this one? I’m planning to redo the react-conformance API which will require breaking changes here too.